byte RECONFIGURE_THRUST_WEIGHT = 0
byte RECONFIGURE_GIMBAL_LINEAR_SOLUTION_DIST_WEIGHT = 1
byte RECONFIGURE_GIMBAL_CURRENT_ANGLE_DIST_WEIGHT = 2
byte RECONFIGURE_GIMBAL_CENTER_DIST_WEIGHT = 3
byte RECONFIGURE_JOINT_TORQUE_WEIGHT = 4
byte RECONFIGURE_ATTITUDE_CONTROL_ROLL_THRESHOLD = 5
byte RECONFIGURE_ATTITUDE_CONTROL_PITCH_THRESHOLD = 6
byte RECONFIGURE_ATTITUDE_CONTROL_YAW_THRESHOLD = 7
